.envrc.template

```diff
@@ -0,0 +1,2 @@
+export API_V3_HOST=
```

Comment:
I updated signs-ui-dev and signs-ui-prod to have their keys. However, as I was doing so, I realized that `_HOST` is not quite correct here, as that would be, e.g., `api-v3.mbta.com`. But the code expects both the host and the scheme. And we don't want to hardcode the scheme of `https` since then we can't swap in `http://localhost:xyz` as an environment variable.

Cora and I recently had a discussion about this on another PR and landed on the term `origin`, which is what the browser calls scheme+host, and I think that's probably the best term. I did a quick search now and can't really find anything better. Better than C#'s "left part" anyway.

So the ENV vars I put in AWS just now were `API_V3_ORIGIN` and `API_V3_KEY`, if you would please update the code here to follow.